main/lsp/LSPTypechecker.cc

```diff
@@ -415,22 +415,11 @@ bool LSPTypechecker::runSlowPath(LSPFileUpdates updates, WorkerPool &workers, bo
     return committed;
 }
 
-void LSPTypechecker::pushDiagnostics(u4 epoch, vector<core::FileRef> filesTypechecked,
-                                     vector<std::unique_ptr<core::Error>> errors) {
+void LSPTypechecker::pushAllDiagnostics(u4 epoch, vector<core::FileRef> filesTypechecked,
```

Comment:
What happens if `foo.rb` is typechecked but has no errors? We should still call `pushDiagnostics`, right?